substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -83,6 +83,10 @@ pub struct CreateCmd {
 	/// errors will be reported.
 	#[arg(long, short = 'v')]
 	verify: bool,
+	/// Chain properties in comma-separated KEY=VALUE format.
+	/// Example: `--properties tokenSymbol=UNIT,tokenDecimals=12,ss58Format=42,isEthereum=false`
+	#[arg(long)]
+	pub properties: Option<String>,
```

Comment:
```suggestion
	pub properties: Vec<String>,
```

Reply:
Changed in [1b8e84a](https://github.com/paritytech/polkadot-sdk/pull/7368/commits/1b8e84a492dd696e0ca9cb9b29296bd5b8695758).